src/renderer/views/Channel/Channel.js

```diff
@@ -12,7 +12,10 @@ import FtSubscribeButton from '../../components/ft-subscribe-button/ft-subscribe
 import ChannelAbout from '../../components/channel-about/channel-about.vue'
 
 import autolinker from 'autolinker'
-import { copyToClipboard, extractNumberFromString, formatNumber, isNullOrEmpty, showToast } from '../../helpers/utils'
+import { copyToClipboard, extractNumberFromString, formatNumber, showToast } from '../../helpers/utils'
+import {
+  isNullOrEmpty,
+} from '../../helpers/strings'
```

Comment:
Could you please also make all of these these imports just a single line, as they are short enough that we don't have to wrap them.